assets/src/edit-story/components/library/panes/media/media3p/media3pPane.js

```diff
@@ -149,20 +142,20 @@ function Media3pPane(props) {
             />
           </ProviderTabSection>
           <Media3pCategories
-            categories={categories.categories}
-            selectedCategoryName={categories.selectedCategoryName}
+            categories={categories}
+            selectedCategoryName={selectedCategoryName}
             selectCategory={selectCategory}
             deselectCategory={deselectCategory}
           />
         </PaneHeader>
         <PaginatedMediaGallery
           providerType={ProviderType.UNSPLASH}
-          resources={media}
-          isMediaLoading={isMediaLoading}
-          isMediaLoaded={isMediaLoaded}
-          hasMore={hasMore}
+          resources={unsplash.state.media}
+          isMediaLoading={unsplash.state.isMediaLoading}
+          isMediaLoaded={unsplash.state.isMediaLoaded}
+          hasMore={unsplash.state.hasMore}
+          setNextPage={unsplash.actions.setNextPage}
           onInsert={insertMediaElement}
```

Comment:
should we use selectedProvider an selectedProviderState here? Then we don't need to extract unsplash above.

Reply:
Coverr will be a separately rendered PaginatedMediaGallery.